programs/openbook-v2/src/instructions/settle_funds.rs

```diff
@@ -5,24 +5,39 @@ use crate::accounts_ix::*;
 use crate::state::*;
 
 pub fn settle_funds<'info>(ctx: Context<'_, '_, '_, 'info, SettleFunds<'info>>) -> Result<()> {
-    let mut acc = ctx.accounts.open_orders_account.load_mut()?;
+    let mut open_orders_account = ctx.accounts.open_orders_account.load_mut()?;
     let mut market = ctx.accounts.market.load_mut()?;
 
+    let mut roundoff_maker_fees = 0;
+
+    if market.maker_fee.is_positive()
```

Comment:
you could use `let mut roundoff_maker_fees = if....` then an `else that returns 0`

Reply:
note that the block is not only assigning the variable but also modifying the state of the market